orders/migrations/0003_static_orders.py

```diff
@@ -0,0 +1,69 @@
+from django.db import migrations, transaction
+from django.utils import timezone
+import datetime
+
+DOG_OWNER_ID = ['4', '5', '6', '7', '8', '9', '10', '11', '12', '13',
+                '14', '15', '16', '17', '18', '19', '20', '21', '22', '23',
+                '4', '5', '6', '7', '8', '9', '10', '11', '12', '13',
+                '14', '15', '16', '17', '18', '19', '20', '21', '22', '23']
```

Comment:
you have duplicates of the id's

Reply:
I wanted the dog owners to have 2 orders each, so I had to duplicate the IDs of the dog owners.
I've changed it to [1, 2, 3...] * 2 for better readability.